package/scripts/precook_test.py

```diff
@@ -0,0 +1,237 @@
+import os
+import time
+import random
+import gc
+import shutil
+
+pid = 1729
+
+
+def make_stat(pid, proc_pid, fixed_value, rand=False):
+    stat_fname = os.path.join(proc_pid, "stat")
+    if rand:
+        with open(stat_fname, "w") as f:
+            # Writing some pid and process name although never used
+            f.write(str(pid) + " ")  # 0 pid
```

Comment:
It's a bit odd to use `write` instead of the much more standard `print()` function. Unless I am missing something can you change to `print()`?